tests/model/test_push_item_content.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pytest import raises
+
+from io import UnsupportedOperation, BufferedReader
+
+from pushsource import PushItem
+
+ITEM_SRC = os.path.join(os.path.dirname(__file__), "data/test_file.txt")
+
+
+def _opener(item):
+    # BuffredReader attributes can't be modified.
+    # So override name in subclass to return None
+    class NoNameBR(BufferedReader):
+        @property
+        def name(self):
+            return None
+
+    return NoNameBR(open(item.src, "rb"))
+
+
```

Comment:
We'll need tests somewhere for the proposed "returns None" case too, e.g. attempting to use `content` on an ErratumPushItem, ContainerPushItem.